File: word_counter_js/lib/word_counter.js
String.prototype.word_counter = function(test_word){
  var words_array = [];
  var word_count = 0;
  words_array = this.split(' ');
  for (var i=0; i < words_array.length; i++){
    if (words_array[i].toLowerCase() === test_word.toLowerCase()){
      word_count++;
    }
  }
  return word_count;

};

File: lib/word_counter.rb
class String
  define_method(:word_counter) do |test_word|
    string = self
    words_array = string.split(' ')
    word_count = 0

    words_array.each do |word|
      if word == test_word
        word_count += 1
      end
    end
    word_count
  end
end
